### PR TITLE
migrate tape to native test runner

### PR DESCRIPTION
**Pull request by zwOk9**

### CODE REVIEW

**Comment by zwOk9** [comment]
Utiliser avec la version 19.8.1
j'ai également utiliser les [assertion](https://nodejs.org/docs/latest-v19.x/api/assert.html)s de node

File: package.json [hunk 1]
```diff
@@ -37,7 +38,6 @@
     "@small-tech/esm-tape-runner": "^2.0.0",
     "@small-tech/tap-monkey": "^1.4.0",
```
Comment (PierreDemailly) [inline package.json:39]:
```suggestion
```
You can remove theses deps

Also, i think we can remove `cross-env` (should not be needed for linter)

File: test/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // Import Third-party Dependencies
```
Comment (PierreDemailly) [inline test/index.js:1]:
```suggestion
// Import Node.js Dependencies
```

**Review by PierreDemailly — COMMENTED** [review]
👍🏻

**Review by fraxken — APPROVED** [review]
LGTM

**Review by PierreDemailly — APPROVED** [review]
Its details but you can also remove the callback params
`test("...", async(t) => {`
-> `test("...", async() => {`

Everything else lgtm

**Pull request merged**